2-library/postges-library/pom.xml

```
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com._6core.lib.java</groupId>
+    <artifactId>postges-library</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

rename package accordingly 
```suggestion
    <groupId>com._6core.lib</groupId>
    <artifactId>reactive-postges-persistent</artifactId>
    <version>0.0.1-SNAPSHOT</version>
```